CONTRIBUTING.md

```diff
@@ -1,41 +1,11 @@
-# How to contribute
-
-👍🎉 First off, thanks for taking the time to contribute! 🎉👍
-
-Check out the [Stellar Contribution Guide](https://github.com/stellar/.github/blob/master/CONTRIBUTING.md) that apply to all Stellar projects.
-
-## Style guides
-
-### Git Commit Messages
+# ⚠️ REPOSITORY DEPRECATED ⚠️
 
-* Use the present tense ("Add feature" not "Added feature")
-* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+**This repository has been moved to: [github.com/stellar/go-stellar-sdk](https://github.com/stellar/go-stellar-sdk)**
 
-### Issues
+All new development will take place there.
```

Comment:
Is the plan to archive this repo to drive this point home?

Reply:
yes, that would be the last notch as will make this repo read-only when it's set to archived in gh repo settings. Should we do that after this merges?